src/overgangsstønad/steg/8-dokumentasjon/SendSøknad.tsx

```diff
@@ -99,7 +101,7 @@ const SendSøknadKnapper: FC = () => {
     const dokumentasjonsbehov = søknad.dokumentasjonsbehov.filter(
       unikeDokumentasjonsbehov
     );
-
+    logDokumetasjonsbehov(dokumentasjonsbehov, ESkjemanavn.Overgangsstønad);
```

Comment:
Vil vi ha tilsvarende for barnetilsyn/skolepenger eller kommer det senere?

Reply:
Jeg legger til for de andre også :)

Reply:
fixd.